### PR TITLE
iOS example now runs on M1 chip

### PR DESCRIPTION
**Pull request by mroloux**
To make our iOS example project run on my new machine, I had to disable Flipper. Don't think we need it anyway - it's a debugging tool:

https://fbflipper.com/

### CODE REVIEW

**Comment by mroloux** [comment]
[Make seatsio-react-native run on M1](https://trello.com/c/VDh1etPs/4491-make-seatsio-react-native-run-on-m1)

File: components/SeatsioSeatingChart.js [hunk 1]
```diff
@@ -37,7 +37,7 @@ class SeatsioSeatingChart extends React.Component {
     injectJsAndReturnDeferredFn(js, transformer) {
         const deferred = new Deferred(transformer)
         const uuid = randomUuid()
-        this.registerPFromise(uuid, deferred)
+        this.registerPromise(uuid, deferred)
```
Comment (bverbeken) [inline components/SeatsioSeatingChart.js:40]: 😱

**Review by bverbeken — COMMENTED** [review]
I'm going to approve, I guess this works, I just can't get it working

**Review by bverbeken — APPROVED** [review]
I'm going to approve, I guess this works, I just can't get it working

File: components/SeatsioSeatingChart.js (hunk 1 above)
Comment (mroloux) [reply components/SeatsioSeatingChart.js:40]: Yeah sorry, that was me 🍍

**Pull request merged**